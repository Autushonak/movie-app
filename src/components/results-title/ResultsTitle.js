import styled from 'styled-components';
import PropTypes from 'prop-types';
import { colors } from '../../constants/colors';

const ResultsTitle = styled.h4`
  font-weight: 900;
  color: ${colors.WHITE};
  margin-right: 10px;
`;

ResultsTitle.propTypes = {
  children: PropTypes.array,
};

export default ResultsTitle;
