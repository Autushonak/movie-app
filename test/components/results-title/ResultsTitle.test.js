import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import ResultsTitle from '../../../src/components/results-title/ResultsTitle';

describe('ResultsTitle ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<ResultsTitle />);
    expect(component).toMatchSnapshot();
  });
});
