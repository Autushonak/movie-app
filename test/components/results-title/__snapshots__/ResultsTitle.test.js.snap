// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsTitle  should be rendered correctly 1`] = `
.c0 {
  font-weight: 900;
  color: #FFFFFF;
  margin-right: 10px;
}

<h4
  className="c0"
/>
`;
